### PR TITLE
fix: fix font name display wrong when config custom font toolbar close #92

### LINKED ISSUE #92: 🐛 [Bug]: 配置工具栏按钮 font，显示的字号名称错误
### Version

latest

### Link to minimal reproduction

工具栏配置：
```
const TOOLBAR_CONFIG = [
  [{ font: [
    'songti',
    'yahei',
    'kaiti',
    'heiti',
    'lishu',
    'mono',
    'arial',
    'arialblack',
    'comic',
    'impact',
    'times',
   ] }],
]
```

实际效果：

image

期望效果：

image

### Step to reproduce

同上

### What is expected

_No response_

### What is actually happening

_No response_

### What is your project name

Wiki

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by kagol**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our Commit Message Guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #92 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Expanded font options in the custom toolbar of the editor, now including a variety of styles such as 'arial', 'comic', and 'times'.

- **Style**
	- Restructured CSS rules for font styles to improve organization without changing existing styles.

### CODE REVIEW

File: packages/fluent-editor/src/assets/font.scss [hunk 1]
```diff
@@ -1,58 +1,61 @@
...
+  .ql-font.ql-picker {
+    width: 108px;
+    span[data-value='arial']::before {
+      content: 'arial';
+      font-family: arial, helvetica, sans-serif;
+    }
+    span[data-value='songti']::before {
+      content: '宋体';
+      font-family: '宋体', 'SimSun';
+    }
```
Comment (zzxming) [inline packages/fluent-editor/src/assets/font.scss:14]: 这样的重复代码用 scss 循环生成好一点
```scss
$font-map: (
  'arial': (
    'arial',
    (arial, helvetica, sans-serif)
  ),
  'songti': (
    '宋体',
    ('宋体', 'SimSun')
  ),
);
@each $font, $font-value in $font-map {
  span[data-value=#{$font}]::before {
    content: nth($font-value, 1);
    font-family: nth($font-value, 2);
  }
}
```

File: packages/fluent-editor/src/assets/font.scss [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
+    span[data-value='mono'].ql-picker-label::before,
+    span[data-value='comic'].ql-picker-label::before,
+    span[data-value='times'].ql-picker-label::before,
+    span[data-value='arialblack'].ql-picker-label::before {
+      width: 60px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
```
Comment (zzxming) [inline packages/fluent-editor/src/assets/font.scss:59]: 为什么只对这几种类型限制超出显示样式呢？直接对所有的加上不是更方便吗

File: packages/fluent-editor/src/assets/font.scss (hunk 1 above)
Comment (kagol) [reply packages/fluent-editor/src/assets/font.scss:14]: 有道理，我后面单开一个PR重构下

File: packages/fluent-editor/src/assets/font.scss (hunk 2 above)
Comment (kagol) [reply packages/fluent-editor/src/assets/font.scss:59]: 历史问题了，我再重构PR中优化下这块代码

**Pull request merged**